packages/api/src/microsoft/teams/api/clients/api_client_settings.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import os
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class ApiClientSettings:
+    """
+    Settings for API clients.
+
+    Attributes:
+        oauth_url: The URL to use for managing user OAuth tokens.
+                   Specify this value if you are using a regional bot.
+                   For example: https://europe.token.botframework.com
+                   Default is https://token.botframework.com
+    """
+
+    oauth_url: str = "https://token.botframework.com"
+
+
+DEFAULT_API_CLIENT_SETTINGS = ApiClientSettings()
+
+
+def merge_api_client_settings(api_client_settings: Optional[ApiClientSettings] = None) -> ApiClientSettings:
```

Comment:
NIT: don't set `= None` here. if you're "merging" something, it doesn't make sense to not pass in an argument